packages/mongo/package.js

```diff
@@ -13,15 +13,15 @@ Package.describe({
 });
 
 Npm.depends({
-  "mongodb-uri": "0.9.7"
+  "mongodb-uri": "0.9.7",
+  "mongodb": "3.6.9"
```

Comment:
I'm not sure if this is not going to included mongodb in the client as well. Are you sure?

Reply:
What client?

Reply:
@harryadel on the app client I think what @filipenevola was meaning here.

Reply:
I'm unsure whether `mongodb` library gets shipped to the client side or not but if it's a concern shouldn't it be a global one? ddp-server package for instance import sockjs which is the server counter part of sockjs-client
https://github.com/meteor/meteor/blob/6f7df8f35f1617b393a1d13c48f2bf70c2be31f6/packages/ddp-server/package.js#L9

Reply:
No, scratch that. @filipenevola has a point.

`ddp-server` as a whole is set to be used in server only 
https://github.com/meteor/meteor/blob/ffcfa5062cf1bf8a64ea64fef681ffcd99fe7939/packages/ddp/package.js#L8